src/main/java/duke/command/CMD.java

```diff
@@ -0,0 +1,12 @@
+package duke.command;
+
+public enum CMD {
```

Comment:
I think enum names should be written in PascalCase based on the coding standard? 🤔